workshops/weather_app/README.md

```diff
@@ -0,0 +1,416 @@
+---
+name: 'Weather App'
+description: 'Making a Weather App using HTML, CSS and JS(no framework involved)'
+author: '@gautamjajoo'
+---
+
+# Weather App
+
+Are you a beginner in JavaScript and don't know where to get started? If yes, you're in the right place! Being a beginner in coding and having to learn the fundamentals can be very painful—but the best way to learn is to make fun, simple projects! In this workshop, you'll build an app that uses web APIs to get the weather for any city.
+
+Here's what the final version looks like:
+
+![main](https://cloud-fw0yxvsf2.vercel.app/0screenshot_from_2020-10-23_19-26-13.png)
+
+The [Source Code](https://github.com/gautamjajoo/BasicWeather) & [Demo](https://Weather.gautamjajoo.repl.co) are hyperlinked here.
+
+By the end of this workshop, you'll have learned how to use web APIs, which you could expand to use even more creatively in future projects. You'll also learn some fundamentals of JavaScript, as well as some nice CSS tricks for making beautiful designs.
+
+## 1. The Prerequisites
+Some basic understanding of HTML, CSS and Javascript/JSON.
+
+If you don't know any of the above, don't worry! This workshop will guide you through it.

```

Comment:
huh? so why are they prerequisites

Reply:
It is better to have a basic understanding of it. If you don't know any of them the workshop will guide through.

Reply:
then why do you need them as a prerequisite?